var/spack/repos/builtin/packages/hipsycl/package.py

```diff
@@ -39,6 +39,9 @@ class Hipsycl(CMakePackage):
     depends_on("python@3:")
     depends_on("llvm@8: +clang", when="~cuda")
     depends_on("llvm@9: +clang", when="+cuda")
+    # hipSYCL 0.8.0 supported only LLVM 8-10:
+    # (https://github.com/AdaptiveCpp/AdaptiveCpp/blob/v0.8.0/CMakeLists.txt#L29-L37)
+    depends_on("llvm@8:10", when="@=0.8.0")
```

Comment:
```suggestion
    depends_on("llvm@8:10", when="@0.8.0")
```

we usually do ranges even for specific versions, cause *sometimes* people use externals with custom suffix, like `0.8.0-thathackedone`